docs/DeveloperGuide.md

```diff
@@ -222,99 +222,55 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 ## **Implementation of main features**
 
-### Add person feature
+## **Implementation of Entity Commands**
+Entity commands include `add`, `delete`, `find`, `clear` commands.
+So `xyzCommand` can be `addPersonCommand`, `addCommandParser` and so on.
 
-#### Implementation
-
-#### Design considerations
-
-<br>
-
-### Edit person feature
-
-#### Implementation
-
-#### Design considerations
-
-<br>
-
-### Delete person feature
-
-#### Implementation
-
-#### Design considerations
-
-<br>
-
-### Find person feature
-
-#### Implementation
-
-#### Design considerations
-
-<br>
-
-### List person feature
-
-#### Implementation
-
-#### Design considerations
+**Step 1**. The user types an `xyz` command in the `CommandBox`, followed by the type of entity `person` or `appt`. This is followed by appropriate arguments and prefixes.
+**Step 2**. The command is passed to the `LogicManager`. `LogicManager` then calls the `AddressBookParser::parseCommand` method to parse the command.
+**Step 3**. The `AddressBookParser` creates a `xyzEntityCommand` object, and call the `xyzCommandParser::parse` method, which is returned to the `LogicManager`.
```

Comment:
It might be good to mention how this may change for certain cases - for example, find appointment uses another layer of parsers